proto/pocket/application/application.proto

```diff
@@ -3,8 +3,12 @@ package pocket.application;
 
 option go_package = "pocket/x/application/types";
 
+import "cosmos_proto/cosmos.proto";
+import "cosmos/base/v1beta1/coin.proto";
+
+// Application defines the type used to store an on-chain definition and state for an application
 message Application {
-  string address = 1; 
-  
+  string address = 1 [(cosmos_proto.scalar) = "cosmos.AddressString"]; // The Bech32 address of the application
```

Comment:
@Olshansk can you add a comment which elaborates on what/why this is and/or references some documentation? I didn't learn much after a 2 min search and GPT convo, I think recording some context here would be helpful in the future, especially in the face of the coming v0.50 migration.

Reply:
https://github.com/cosmos/cosmos-proto/blob/b6a88f6e0255c2fb92f5d9a7211caa38769e33d5/proto/cosmos_proto/cosmos.proto#L66

This may clear things up a little. Basically this just defines at the protobuf layer the string is using the encoding scheme of the `cosmos.AddressString` (bech32) type. Unsure around how much this is enforced.